El login y el registro de usuarios esta funcional

Carolinasaja opened the pull request:
Registro de usuario funcional conectado a la BD y login funcional

nrocha02 committed:
Imágenes consultadas desde la DB

nrocha02 committed:
Dashboard interactúa con las imágenes para añadirlas y eliminarlas en las operaciones de los productos

nrocha02 committed:
El slug se genera de manera automática para las categirías, el usuario no interviene

nrocha02 committed:
Arreglado las imágenes del carrito

nrocha02 committed:
fallback detalle producto

nrocha02 committed:
producto fix

Carolinasaja committed:
El login y el registro de usuarios esta funcional

Carolinasaja committed:
El login y el registro de usuarios esta funcional

nrocha02 reviewed (changes requested):
- No está vinculado el botón de inicio de sesión en el index
- Hay que cambiar el template de login para que en la barra diga "email"
- Hay que modificar la función de login para que solamente revise el correo
- Idealmente añadir alguna clase de confirmación visual al lograr un login exitoso

nrocha02 committed:
Imagenes productos (#189)

* Imágenes consultadas desde la DB

* Dashboard interactúa con las imágenes para añadirlas y eliminarlas en las operaciones de los productos

* El slug se genera de manera automática para las categirías, el usuario no interviene

* Arreglado las imágenes del carrito

* Arreglado template detalle producto

Carolinasaja committed:
Actualizacion login:
-Boton index redirigido al inicio de sesion
-Link para registrarse redirige a registro de usuarios
-Barra y auth solo con email

Carolinasaja committed:
Actualizacion login:
-Boton index redirigido al inicio de sesion
-Link para registrarse redirige a registro de usuarios
-Barra y auth solo con email

nrocha02 committed:
Eliminado proyecto FastAPI y actualización de script para crear la base de datos

nrocha02 committed:
Documentación del desarrollo

nrocha02 committed:
Key del proyecto movida al env

nrocha02 committed:
Añadido dependencia y cambios en settings para uso en producción

nrocha02 committed:
Añadido dependencia gunicorn

nrocha02 committed:
Cerrar sesión y confirmaciones visuales

nrocha02 committed:
Cerrar sesión y confirmaciones visuales

nrocha02 committed:
Merge branch 'login' of https://github.com/nrocha02/2025_MA_CAPSTONE_704D_GRUPO_2 into login

nrocha02 committed:
urls fix

nrocha02 reviewed (approved):
Ya están las mejoras incorporadas y también se actualizó la rama con los cambios nuevos del main.
